spec/controllers/allocations_controller_spec.rb

```diff
@@ -100,7 +103,7 @@
           end
 
           it 'doesnt mess up the allocation history updated_at because we surface the value' do
```

Comment:
This test seems less relevant now that COM name isn't stored in the Allocation record.